schemas/Error.json

```diff
@@ -3,17 +3,20 @@
   "description": "Triggered when an error occurred",
   "properties": {
     "eventName": {
+      "title": "EventNameError",
       "enum": ["Error"]
     },
     "domain": {
       "type": "string",
+      "title": "ErrorDomain",
       "enum": [
         "E2EE",
         "VOIP"
       ]
     },
     "name": {
       "type": "string",
+      "title": "ErrorName",
```

Comment:
This and and the one on the domain were to reduce potential name clashes in the future.